scripts/spack/packages/umpire/package.py

```diff
@@ -1,275 +1,25 @@
-
-        if '+rocm' in spec:
-            entries.append(cmake_cache_option("ENABLE_HIP", True))
-            entries.append(cmake_cache_path(
-                "HIP_ROOT_DIR", '{0}'.format(spec['hip'].prefix)))
-            archs = self.spec.variants['amdgpu_target'].value
-            if archs != 'none':
-                arch_str = ",".join(archs)
-                entries.append(cmake_cache_string(
-                    "HIP_HIPCC_FLAGS", '--amdgpu-target={0}'.format(arch_str)))
-            # BEGIN SERAC EDIT
-            # Fix blt_hip getting HIP_CLANG_INCLUDE_PATH-NOTFOUND bad include directory
-            if self.spec.satisfies('%clang') and 'toss_4' in self._get_sys_type(spec):
-                hip_root = spec['hip'].prefix
-                rocm_root = hip_root + "/.."
-                clang_version= str(self.compiler.version)
-                hip_clang_include_path = rocm_root + "/llvm/lib/clang/" + clang_version + "/include"
-                if os.path.isdir(hip_clang_include_path):
-                    entries.append(cmake_cache_path("HIP_CLANG_INCLUDE_PATH", hip_clang_include_path))
-
-            # END SERAC EDIT
-        else:
-            entries.append(cmake_cache_option("ENABLE_HIP", False))
-
-        return entries
+    depends_on("blt@0.5.1:", type="build", when="@2022.03.0:")
 
     def initconfig_package_entries(self):
+        entries = BuiltinUmpire.initconfig_package_entries(self)
         spec = self.spec
-        entries = []
-
         option_prefix = "UMPIRE_" if spec.satisfies("@2022.03.0:") else ""
-
-        # TPL locations
-        entries.append("#------------------{0}".format("-" * 60))
-        entries.append("# TPLs")
-        entries.append("#------------------{0}\n".format("-" * 60))
-
-        entries.append(cmake_cache_path("BLT_SOURCE_DIR", spec['blt'].prefix))
-        if spec.satisfies('@5.0.0:'):
-            entries.append(cmake_cache_path("camp_DIR", spec['camp'].prefix))
-        entries.append(cmake_cache_option(
-            "{}ENABLE_NUMA".format(option_prefix), '+numa' in spec))
-        entries.append(cmake_cache_option(
-            "{}ENABLE_OPENMP".format(option_prefix), '+openmp' in spec))
-        entries.append(cmake_cache_option(
-            "ENABLE_BENCHMARKS", 'tests=benchmarks' in spec))
-        entries.append(cmake_cache_option(
-            "{}ENABLE_EXAMPLES".format(option_prefix), '+examples' in spec))
         entries.append(cmake_cache_option(
-            "{}ENABLE_DOCS".format(option_prefix), False))
-        entries.append(cmake_cache_option("BUILD_SHARED_LIBS", '+shared' in spec))
-        entries.append(cmake_cache_option("ENABLE_TESTS", 'tests=none' not in spec))
-    # BEGIN SERAC EDIT
-        entries.append(cmake_cache_option(
-            "{}ENABLE_DEVICE_ALLOCATOR".format(option_prefix), '+device_alloc' in spec))
-    # END SERAC EDIT
+            "{}ENABLE_DEVICE_ALLOCATOR".format(option_prefix), "+device_alloc" in spec))
```

Comment:
This shouldnt be needed now:

https://github.com/spack/spack/blob/f802b64e7a6071f36c928150351c1ac30bc83d4b/var/spack/repos/builtin/packages/umpire/package.py#L233-L234

Don't rebuild because of this.